styles/styles.css

```
@@ -0,0 +1,367 @@
+  border: 2px solid #e5e5e5;
+  border-radius: 4px;
+}
+
+.item-info h3 {
+  font-size: 24px;
+  line-height: 28px;
+  font-weight: 700;
+  color: #000000;
+}
+
+a {
+  text-decoration: none;
+}
+
+.container {
+  font-size: 18px;
+  line-height: 21px;
+  font-weight: 400;
+  color: #333333;
+}
+
+.more-details-button,
+.added-to-fav-button,
+.add-to-fav-button {
+  font-size: 16px;
+  line-height: 20px;
+  font-weight: 700;
+  padding-left: 26px;
+  padding-right: 26px;
+  padding-top: 8px;
+  padding-bottom: 8px;
+  color: #ffffff;
+  border: none;
+  border-radius: 4px;
+}
+
+.more-details-button {
+  background-color: #756157;
+}
+
+.add-to-fav-button {
+  background-color: #82b3d3;
+}
+
+.added-to-fav-button {
+  background-color: #7db54f;
+}
+
+.item-rating p {
+  position: absolute;
+  font-size: 16px;
+  line-height: 20px;
+  padding-left: 26px;
+  padding-right: 26px;
+  padding-top: 8px;
+  padding-bottom: 8px;
+  color: #333333;
+  border: none;
+  border-radius: 4px;
+  background-color: #f2f2f2;
+}
+
+.pagination-item:nth-child(1) a {
+  font-size: 20px;
+  line-height: 36px;
+  font-weight: 700;
+  color: #000000;
+  background-color: #f2f2f2;
+  border-radius: 4px;
+}
+
+.pagination-item a {
+  font-size: 20px;
+  line-height: 36px;
+  font-weight: 700;
+  color: #ffffff;
+  background-color: #82b3d3;
+  border-radius: 4px;
+}
+
+.button-want-here,
+.hotel-search-button {
+  text-decoration: none;
+  background-color: #756157;
+  color: white;
+  border-radius: 4px;
+  font-weight: 700;
+}
+
+.button-want-here {
+  padding-top: 8px;
+  padding-bottom: 8px;
+  padding-left: 34px;
+  padding-right: 34px;
+}
+
+.hotel-search-button {
+  padding-top: 8px;
+  padding-bottom: 8px;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.background-image-sedona {
+  background-image: url("../images/index-page-background.jpg");
+  background-size: 100% auto;
+  background-repeat: no-repeat;
+}
+
+.description-sedona {
+  font-size: 30px;
+  font-weight: 700;
+  line-height: 36px;
+  text-align: center;
+}
+
+.description-reasons {
+  font-size: 22px;
+  line-height: 26px;
+  font-weight: 400;
+  text-align: center;
+  color: #333333;
+}
+
+.advantages-item {
+  text-align: center;
+}
+
+.advantages-description {
+  font-size: 18px;
+  line-height: 21px;
+  font-weight: 400;
+}
+
+.advantages-title {
+  font-size: 24px;
+  line-height: 28px;
+  font-weight: 700;
+}
+
+.advantages-item:nth-child(4n+1) {
+  color: #ffffff;
+}
+
+.advantages-item:nth-child(4n+1) {
+  background-color: #82B3D3;
+}
+
+.advantages-item:nth-child(2),
+.advantages-item:nth-child(4) {
+  background-color: rgba(131, 179, 211, 0.12);
+}
+
+.advantages-item:nth-child(3n+3),
+.advantages-item:nth-child(8) {
+  background-color: rgba(131, 179, 211, 0.2);
+}
+
+.hotel-search {
+  text-align: center;
+}
+
+.hotel-search-header {
+  font-size: 30px;
+  line-height: 36px;
+}
+
+.hotel-search-contents {
+  font-size: 22px;
+  line-height: 26px;
+}
+
+.subscribtion {
```

```suggestion
.subscription {
```
в разметке тоже нужно исправить